src/main/java/mekanism/common/tile/TileEntityGasTank.java

```diff
@@ -421,4 +423,26 @@ public TileComponentSecurity getSecurity()
 	{
 		return securityComponent;
 	}
+
+	@Override
+	public String[] getMethods()
+	{
+		return methods;
+	}
+
+	@Override
+	public Object[] invoke(int method, Object[] arguments) throws Exception
+	{
+		switch(method)
+		{
+			case 0:
+				return new Object[] {gasTank.getMaxGas()};
+			case 1:
+				return new Object[] {gasTank.getStored()};
+			case 2:
+				return new Object[] {gasTank.getGas()};
+			default:
+				throw new NoSuchMethodException();
```

Comment:
OC handles it correctly and here is similar: https://github.com/aidancbrady/Mekanism/blob/master/src/main/java/mekanism/common/tile/TileEntityLaserAmplifier.java#L364